### PR TITLE
Wrote initial draft of 2.8: logging

### LINKED ISSUE #216: 2.8 - Process Logging

### PR DESCRIPTION
**Pull request by Duckapple**

### CODE REVIEW

File: report/28_Process-Logging.md [hunk 1]
```diff
@@ -1 +1,23 @@
-What do you log in your systems and how do you aggregate logs?
+## Process Logging
+
+Logging output from processes is important in order to know what is going on in the system. If your program in no way outputs any information, then it is hard to diagnose faults or detect attacks, which can make fixing those things harder.
```
Comment (Jakobis) [inline report/28_Process-Logging.md]: "a program" i stedet for "your program"

File: report/28_Process-Logging.md [hunk 2]
```diff
@@ -1 +1,23 @@
...
+
+We did not actually set up any visualizations or use the logs for anything, however. We made it work for illustration purposes, but in part due to unfamiliarity with the Kibana interface, there were never any visualizations set up of the data that had been logged. When writing this report, some visualizations were made and saved, and finding features in the UI such as the save button location, setting up x- and y-axes and setting the timespan for queries were not necessarily difficult, but not straight-forward. If Kibana had more focus on visualization as code, then we developers might have been more interested.
```
Comment (Jakobis) [inline report/28_Process-Logging.md]: Illustration purposes?

**Review by Jakobis — DISMISSED** [review]

**Comment by Duckapple** [comment]
closes #216

File: report/28_Process-Logging.md [hunk 3]
```diff
@@ -1 +1,38 @@
...
+We log all API call events, including but not limited to:
+
+- `/latest`.
+- Getting messages by all/specific user(s), along with a count of returned messages.
+- Posting twits, both for the successful and failure paths.
+- Registering a user, including cases with mail collisions and the like.
+- Getting messages for follows, including count (again).
+- Any of the calls made when unauthorized.
+
+This logging is done all with the information level, and it makes sense:
```
Comment (Hjaltesorgenfrei) [inline report/28_Process-Logging.md]: Does it make sense? Also Information level needs some context.

**Review by Hjaltesorgenfrei — CHANGES_REQUESTED** [review]

File: report/28_Process-Logging.md (hunk 3 above)
Comment (tobloef) [reply report/28_Process-Logging.md]: Yeah, der er nok noget der skal omskrives her, det der kolon er også lidt weird.

**Review by Hjaltesorgenfrei — APPROVED** [review]

**Pull request merged**